src/BonsaiBalance.Mobile/BaseProject/tailwind.config.js

```diff
@@ -0,0 +1,22 @@
+/** @type {import('tailwindcss').Config} */
+
+const colors = require('tailwindcss/colors')
+
+const customTheme = {
```

Comment:
How is `customTheme` being used? Not sure if I understand this.

Reply:
You can use build themes that are collections of colors (a good use case is our app having a light/dark theme). It will make more sense once I implement it on some components. I just picked some arbitrary colors for demonstration purposes